R/fda-table_20.R

```diff
@@ -0,0 +1,136 @@
+# Pre-processing
+
+library(dplyr)
+library(scda)
+library(tern)
+
+adsl <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "adsl") %>%
+  df_explicit_na()
+
+adae <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "adae") %>%
+  mutate(
+    aesev = factor(
+      case_when(
+        AESEV == "MILD" ~ "Mild",
+        AESEV == "MODERATE" ~ "Moderate",
+        AESEV == "SEVERE" ~ "Severe"
+      ),
+      levels = c("Severe", "Moderate", "Mild")
+    ),
```

Comment:
I don't think we need to reformat the text here since the uppercase levels are still easy to understand.

```suggestion
```